### ISSUE TITLE
Rename the About repo to help users better anticipate and understand the content it contains

### ISSUE THREAD

**Issue by ElenaFdR** [publiccodenet/about#782]
We've received feedback from casually interested users that navigating to About by accident is a baffling and overwhelming user experience.

Most websites' 'About' is a potted history of the organization, probably accompanied by a vision or photos of the founders/team. Our About is a blueprint for how to run another Foundation, our canonical source of truth generated by making [documentation central to our process](https://about.publiccode.net/activities/documentation/).

I propose we rename the About repo to better reflect its size, scope and ambition. 

3 suggestions have been made so far:

- howto.publiccode.net (riffing on the traditional HOWTO file once bundled with codebases)
- documentation.publiccode.net (inspired partially by the Standard for Public Code insisting on appropriate documentation of a project)
- meta.publiccode.net (inspired partially by https://meta.decidim.org/)

**Comment by Ainali** [comment]
Another idea is to call it readme.publiccode.net because it is kind of a thing you should read before running an organization like this.

**Comment by ElenaFdR** [comment]
@Ainali I have a concern that naming the repo README creates a similar expectation misalignment as it being called About, because README is also a well known concept with an established form (namely, being a single file). Most repos named 'README' on Github are templates for README files.

**Comment by Ainali** [comment]
@ElenaFdR That's a very good point!

**Comment by lscheske** [comment]
I like meta & How-to... Would a poll not be easier for this?

**Comment by bvhme** [comment]
The 'about' site intends to offer us and those who are interested in our organization and how we operate both:

* answers to their questions 
* as well as the opportunity to explore our workings and 
* even contribute to it.

Versus publiccode.net which is _spiel_.

This content being the 'about' section of our website is a deliberate choice. It serves to make sure that the journey into the information and inner workings of our organization is always an unbroken one.

I see this as the core of us being an 'open' organization. Being an open organization means operating with your guts on the table.

Putting our innards so center stage is a mechanism that holds us accountable to our ability to explain what we're doing to each other and the rest of the world, as we ask this of everyone we work with and should lead by example. The fact that this issue came up is, to me, a testament to that working.

I do not think we need to fulfill others' expectations of an 'about' page. If we can offer them what they want and more, that would be better.

I do however, feel we should try to fulfill those people's needs. So if we need the 'about' home page to have 'a potted history of the organization, probably accompanied by a vision or photos of the founders/team', then maybe we should go ahead and add that.

This, however, should not stand in the way of us enabling anyone who would like to from exploring our organization and activities.

I think the solution to us not being able to clearly explain something should not be to obscure that thing. I don't think you can solve the problem this issue highlights by adding complexity, and I believe that will only make it worse.

The issue raised here is an inditement of the content we offer more than the fact that we offer people information about our organization and activities on the 'about' link or how we name that.

I would also like to signal that I fear us undermining our values and principles and our ability to deliver on our ambitions. I think that changing the role of this codebase is choosing to avoid solving a problem that is core to our future success.

**Comment by angelplasma** [comment]
I empathize with the overall goal Boris surfaces (fusing radical transparency with a whole-hearted invitation to collaborate), and get the sense that this core idea resonates with the group—this is one of the things I love about how the foundation operates. Here is my perspective on setting and meeting expectations in general, and specifically for "About".

**TL;DR**
"About" literally means "in reference to"—by definition, it is a reflection of the thing, not the thing itself. On a website, people click "About" to find an overview, not the inner workings (i.e. nearly the opposite). Inviting people to dive in is _great_, but the invitation can easily follow or sit alongside a potted history. Crucially: Building trust requires meeting (and then exceeding) expectations, while moments of confusion diffuse the sense that the foundation might be the thought leader capable of bringing people together to change the world.

**I see two basic failures with the About section, as is:**
1. Users don't get the story they were likely expecting
2. The invitation to explore is hidden behind the wrong word

**I see two basic solutions:**
1. Lead with the potted history, and follow it with the invitation
2. Change the name to something like "How to", and save the history for another page (e.g. Careers, which already has an 'About us' subsection—arguably, that page should change to "Our Organization" or "Company", etc)

I like solution #2 for how it creates clearer targets for audiences (as well as cleans up the existing redundancy on Careers). **The guts are still on the table.** I think solution #1 helps, but doesn't address the second failure listed above.


**Further thoughts**
Good communication (in our context) does more than broadcast—it connects, building upon mutual understanding to create trust and a sense of partnership, establish a dialogue, and excite collaboration. Naming and visual conventions are useful in leveraging people's existing mental models, and meeting their expectations helps open the door to new concepts to be presented later. Conversely, subverting expectations—say, by using words to refer to non-standard concepts—can destabilize, and lend the feeling of not knowing which way is up.

To move people, it helps to first show them that they are seen. This holds true in both minor and major situations. Making a logical + emotional connection involves meeting them where they are—and then like any good organization who wishes to change the way people think about the world, we have to take them on a journey. (Aside: In the case of publiccode.net, this is actually many little journeys, for many different people.)

When it comes to making or breaking expectations:

In making its case, publiccode.net's general tone tends to be _creative_, more than _argumentative_. This additive, relaxed approach is best at introducing _new_ terms and ways of thinking, which can be effective at creating new mental spaces to inhabit, after which old conceptions can fade as people circle back to redefine and refactor things for themselves.

It's fine to confront pre-existing notions directly, but this should involve explicit arguments that work to change someone's mind—structured, intentional conflict which attempts to force a suspension of disbelief, so transformation (hopefully) happens in the moment via the questioning, disassembling and abandonment of old notions.

To be confrontational, best to be explicit. Greet them with a quick little story, but then speak to what the foundation is _really_ about—that what you really want for them is to be a part of it, or even spin up a new instance. But I would argue against a simple bait-and-switch. **My personal experience of clicking on About** was that I began to get a sense of what the foundation is, before I was ushered toward what felt like a hall of mirrors. It was as though there was no _there_ there—no people or history or context, just more process. It seemed good-natured, but was genuinely disorienting. Instead of a inspirational tale about who runs the site, I was presented with what feels geared toward a search index bot.

People like a good story, _and_ they need an org that can clearly lead them toward a better future. Let's give em both ;)

(I can't believe you read all that. https://youtu.be/T1XgFsitnQw)

**Comment by bvhme** [comment]
I very much agree with all of that sentiment. And I understand where the proposed solution comes from.

I do however feel that the proposed solution is a dis-integration of our content and creates a separation between the way we present ourselves and how we operate. I understand that this is common practice in the _communications_ field, however I think that it goes against what we want to be.

The people we're trying to work with trust us because:

* we are experts in open working, and 
* because we actually do things over just talking about it

The About section is what you described, it __is__ the digital reflection of our organization.

As we don't have a clear high level description about ourselves or history formulated this is not reflected there.

It is not a section on how to run _a_ Foundation for Public Code. It is a section on how _the_ Foundation for Public Code runs. It is the codebase of the Foundation for Public Code, it is the instance. And it's the codebase we want all involved parties to contribute to.

Of course this does not mean that the problem is not real. And it needs to be addressed. However burying the very things which people (should) trust us for because its presentation is confusing and complex cannot be the solution.

**Comment by demkodo** [comment]
I think we may need to involve the whole team and take a poll. Especially if as you say "this is a section on how the Foundation for Public Code runs", your use of 'how' actually makes a strong case for "how to". I think the issue we have here is the insistence on your interpretation on "About" in the context of our work being the only right interpretation, or simply put a 'your way or the highway' kinda sentiment :)

**Comment by lscheske** [comment]
I really like @angelplasma's interpretation and expert proposal on finding a good solution for our information infrastructure. 

I disagree with you @bvhme that that solution goes against what we want to be. I think it actually augments our content because:

a) our community, potential community, and staff will find things more easily
b) it provides more accurate descriptions of what the different components offer
c) catering somewhat to what people expect will give them the feeling that we are empathic and understanding of what our community needs/wants --> this in turn builds trust in us (as Ryan explained more eloquently)
d) if people understand what we are about, and are led in a more effective & efficient way to places they can contribute to, we will be more welcoming to contributions.

My no-fluff bottom line: we should trust the content (@ElenaFdR) and design (@angelplasma) experts that are tasked to make our work understandable, accessible, and open.

**Comment by bvhme** [comment]
I very well understand the benefits that come from solving this issue. It is not as if this is not a frustration for me as well.

However, the proposed solution is a fundamental change in the way our organization works. Decoupling our presentation from our operations and decoupling engagement from participation in our communications. I see serious organizational and operational risks and would like to see those addressed or at least covered for.

Every change always comes with costs and benefits. The proposed solution both comes with significant costs that I don't see addressed and is very hard to reverse in the future.

I understand that I might not have been able to articulate these risks clearly enough as well as describe their costs. I will try to do this so that we can get a more complete image before so that we can be sure we'll be able to move forward. I'll try to prepare some posts to explain this.

This proposed solution however is not the only possible solution and there are easily made tweaks to the current content that can alleviate some of these problems before a change like this is made. I would rather see us work iteratively, that will also make a major change like this easier.

**Comment by nebster** [comment]
I also believe we would be better served by at least a name that reflected the purpose of the information currently labeled "About" and am unsure, @bvhme, why you believe a change to this name would be so disruptive.  I agree that "how to" does not describe the fact that this is the actual currently "running state" of the Foundation as it is, so perhaps "Operating Details" or "Operational Index", "Operational Directory", "Operating Documentation", or perhaps "Functional Foundation Details".  I understand that you do not want to create totally separate bins for representation and deeper understanding, but I think it would make sense to illustrate in our own work the difference between source and application.   I am not sure that "About" is canonical in some way such that by changing one name, we break the paradigm of open working like you seem to suggest?

**Comment by bvhme** [comment]
I think I'm not seeing some larger strategy that is behind this and I think I'd like to understand that a bit more. 

For now I'll just try to make a bit more clear why I respond like this and what my fears are.

To be clear off the bat, **I'm not at all opposed to renaming this repository**. Due to how I interpreted this issue I read it as a much larger thing, changing the role of the content of the repository in our presentation, enforced through changing the name of this repository.

## Who are engaging and participating

Since we're an open organization the people we're making content for is a very wide gammut and the people that are making content is also a very wide gammut.

<img width="391" alt="Legend" src="https://user-images.githubusercontent.com/1917629/99669067-d4ce6f80-2a6e-11eb-820d-f06ad59b8d34.png">

On one end of the spectrum there are the 'new users' we'd like to reach. This audience is a mix of public technologists, public management and public policy people as well as business people. These are groups with greatly varying jargons and priorities. This audience will likely interact sparsely but those interactions will be of very high value.

On the other side of the spectrum are the people who interact with the content on a daily basis. Most of these will be maintainers (people with a responsibility towards maintaining the content or it's community).

In the middle are all the people that have interactions with us that we're trying to help contribute and become more of a part of our community.

In the essence everyone employed by the Foundation for Public Code is both a communication as well as a community person. This of course doesn't mean that other people in our community are not communications and community people, perhaps even in the context of the Foundation for Public Code. 

When I talk about maintainers I mostly mean staff for now, and they also make up a lot of the community. Hopefully we can move this needle in the future.

## An ideal interface

I believe we should strife to build a web presence which is an integrated whole including both the targetted interventions for specific new users (think membership page, team page and careers page), more general content (our homepage, governance) and working processes (community calls).

<img width="391" alt="What I think it should be" src="https://user-images.githubusercontent.com/1917629/99667378-9a63d300-2a6c-11eb-8664-f036eac3f8ff.png">

Ideally any interaction helps others achieve their goals as well as meaningfully contribute to ours.

A good open organization would allow new people to dip in their toe and if they're interested pull them in and transform them into a contributor.

In any case making sure we have a gradient here, or a user journey that is uninterupted, is crucial to enabling the organization for people to feel both informed and co-invested. As well as making sure that information people actually receive is up to date and we can easily fulfil our commitments.

## How I see it now

Our current 'website' has mainly two sections. One of these contains only targetted interventions for specific audiences whilst the other contains all of the information about our organization.

*(There is also a projects section, which I'm going to omit for this as I think it's not that important for this discussion)*

This came about in order to be able to be able to craft content fully with custom design etcetera for goal oriented content on publiccode.net whilst we kept the ease of contributing as low as possible on About.

These two are not very integrated and operate almost separately as a result.

<img width="391" alt="How I see it now" src="https://user-images.githubusercontent.com/1917629/99667383-9c2d9680-2a6c-11eb-86f5-302731eef4c7.png">

Joining the website about our organization to the other sites is the 'About' link.

This decision was made to make sure the content about our organization would always be a integral part of our presentation in the hope that that would force us to provide that gradient where one can ease into the rabit hole when they're ready. Combining all of our information in 1 repository meant that it was easy to understand and contribute to for maintainers.

And a lot of about pages of organizations I'd naturally compare us to feature very similar content, like the [United Nations](https://www.un.org/en/about-un/), [European Union](https://europa.eu/european-union/about-eu_en), [Apache Foundation](https://www.apache.org/foundation/), [Wikimedia Foundation](https://wikimediafoundation.org/about/), with organizational and process information. My hope was that we could go as far as the Apache Foundation and really make sure that information was also canonical.

We've apparently not been able to provide this quality of the content or have not been able to get it up to par.

## What I'm afraid off

I'm afraid that by removing or weakening the interface between our information that we will fall in the trap of on one side building a stunning introduction and on the other side a documentation platform.

<img width="391" alt="What I'm afraid of" src="https://user-images.githubusercontent.com/1917629/99667380-9b950000-2a6c-11eb-8901-c52812c82aa0.png">

This is a situation we've always been naturally pulled towards since it is easier for operations, a contributor doesn't need to think about others when contributing, and for communications, as this provides fine-grained control over output and messaging.

I however see this logically leading to both being out of date and reflecting different aspects of our organization. Next to that they'll both become very hard to contribute to as the documentation platform decends into specialised processes and jargon and the website into high-detail precision interventions.

So this situation makes contributors immediate goals easier to achieve whilst structurally eroding the open organization. A very dangerous state to end up in.

## What I would like to know

So, in conclusion, I would like to know:

* if we make a change like the proposed one is how the integration or link will be between content
* I would like to know what this changes to the way we will be developing content with our entire community
* How do we prevent sources with different goals from drifting apart and disconnecting

Thanks!

**Comment by nebster** [comment]
This is great, @bvhme.  I think we are all quite a bit closer to being on the same page than we thought.  I am not sure how the conversation turned to this larger structural relationship between these functional pages and the "outreach" main site, but I interpreted this issue as being very specifically about the inaccuracy of the name 'About'.  I believe that name is a mischaracterization of the depth and breadth and operational primacy of the pages therein, qualities that you are trying rightly to defend.

As far as the issue of tighter integration between the outreach/comms site and the documentation, which you have so excellently illustrated: if I am not mistaken, in their work on the main site redesign @angelplasma and @ElenaFdR are focusing on centering and highlighting the documentation materials more thoroughly throughout.  In fact, it was in this very integrative activity where the cognitive dissonance produced by calling these canonical documents 'About' became ever more apparent.  To be honest, I think some of the problem comes from our use of flat file Jekyll sans javascript, because with all those sites like Apache and EU that you mention, every single About link on the homepage is a dropdown menu that indicates how rich the content is there to be explored.  Instead of going there, we could instead bubble up the first level of About page links into our mainsite menuing:  Organization, Activities, Glossary, like you suggested in our chat?

**Comment by bvhme** [comment]
Just as an FYI, the changes that I was arguing against have happened in the meantime, so I hope that the arguments I've put forward before can have a place in changes going forward. This means that not renaming the section of our website that's hosted in this repository is no longer a concern for me.

**Comment by clausmullie** [comment]
I'd like to propose closing this issue for now. The website has been reconfigured, the about link moved down, and hyperlinked with the text 'how we work'. With that, it seems a middle way was found between the original intent behind this issue, and the points raised during the discussion. This thread still contains a lot of very valuable thinking (great to have it in the open!) and we will no doubt revisit this at some point, but it does not offer anything actionable for now, and so my sense is that it should be closed. What do you think @bvhme ?

**Comment by clausmullie** [comment]
Closing this issue, as per comment above
